### DISCUSSION TITLE
Help connecting to remote XML directory

### DISCUSSION THREAD

**Discussion by mtiesman** [PaloAltoNetworks/pan-os-php#689] [Q&A]
Hello all. I am working to get Docker configured to use the PAN-OS-PHP. Our XML files are located on a remote server and directory. Can someone help me with the command to allow me to connect to the remote directory so I can then access my XML files? Thanks for reading.

**Comment by swaschkut** [comment]
one option would be:
at the system where you are planning to run the docker container,
you also need to connect your remote server directory to the host.
via cli go to the host directory where your remote server file store is connected to and run the docker command

docker run --name panosphp --rm -v ${PWD}:/share -it swaschkut/pan-os-php:latest

as I am not a docker expert, there are for sure other options, but I am not aware of these
